extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -41,6 +37,21 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined):
 		return;
 	}
 
+	// 2. select input or output binding
+	const inputOutputItems: string[] = [constants.input, constants.output];
+
+	const selectedBinding = (await vscode.window.showQuickPick(inputOutputItems, {
+		canPickMany: false,
+		title: constants.selectBindingType,
+		ignoreFocusOut: true
+	}));
+
+	if (!selectedBinding) {
+		return;
+	}
+
+	const bindingType = selectedBinding === constants.input ? BindingType.input : BindingType.output;
```

Comment:
Usually for stuff like this I find it easier to make the original quick pick items contain the actual type I want. e.g. https://github.com/microsoft/azuredatastudio/blob/main/extensions/sql-database-projects/src/dialogs/publishDatabaseQuickpick.ts#L103